src/syft/lib/pandas/__init__.py

```diff
@@ -0,0 +1,199 @@
+        ("pandas.Series.add", "pandas.Series"),
+        ("pandas.Series.sub", "pandas.Series"),
+        ("pandas.Series.mul", "pandas.Series"),
+        ("pandas.Series.div", "pandas.Series"),
+        ("pandas.Series.truediv", "pandas.Series"),
+        ("pandas.Series.floordiv", "pandas.Series"),
+        ("pandas.Series.mod", "pandas.Series"),
+        ("pandas.Series.pow", "pandas.Series"),
+        ("pandas.Series.radd", "pandas.Series"),
+        ("pandas.Series.rsub", "pandas.Series"),
+        ("pandas.Series.rmul", "pandas.Series"),
+        ("pandas.Series.rdiv", "pandas.Series"),
+        ("pandas.Series.rtruediv", "pandas.Series"),
+        ("pandas.Series.rfloordiv", "pandas.Series"),
+        ("pandas.Series.rmod", "pandas.Series"),
+        ("pandas.Series.rpow", "pandas.Series"),
+        ("pandas.Series.lt", "pandas.Series"),
+        ("pandas.Series.gt", "pandas.Series"),
+        ("pandas.Series.le", "pandas.Series"),
+        ("pandas.Series.ge", "pandas.Series"),
+        ("pandas.Series.ne", "pandas.Series"),
+        ("pandas.Series.eq", "pandas.Series"),
+        ("pandas.Series.argsort", "pandas.Series"),
+        ("pandas.Series.round", "pandas.Series"),
+        ("pandas.Series.head", "pandas.Series"),
+        ("pandas.Series.tail", "pandas.Series"),
+        ("pandas.Series.any", "syft.lib.python.Bool"),
+        ("pandas.Series.shape", "syft.lib.python.Tuple"),
+        ("pandas.Series.all", "syft.lib.python.Bool"),
+        ("pandas.Series.argmax", "syft.lib.python.Int"),
+        ("pandas.Series.nbytes", "syft.lib.python.Int"),
+        ("pandas.Series.mean", "syft.lib.python.Float"),
+        ("pandas.Series.ndim", "syft.lib.python.Int"),
+        ("pandas.Series.size", "syft.lib.python.Int"),
+        ("pandas.Series.hasnans", "syft.lib.python.Bool"),
+        ("pandas.Series.empty", "syft.lib.python.Bool"),
+        ("pandas.Series.T", "pandas.Series"),
+        ("pandas.Series.dropna", "pandas.Series"),
+        ("pandas.Series.to_frame", "pandas.DataFrame"),
+        ("pandas.Series.to_list", "syft.lib.python.List"),
+        (
+            "pandas.Series.sum",
+            UnionGenerator["syft.lib.python.Float", "syft.lib.python.Int"],
+        ),
+        ("pandas.Series.median", "syft.lib.python.Float"),
+        (
+            "pandas.Series.max",
+            UnionGenerator[
+                "syft.lib.python.Bool", "syft.lib.python.Float", "syft.lib.python.Int"
+            ],
+        ),
+        (
+            "pandas.Series.min",
+            UnionGenerator[
+                "syft.lib.python.Bool", "syft.lib.python.Float", "syft.lib.python.Int"
+            ],
+        ),
+    ]
+
+    add_modules(ast, modules)
+    add_classes(ast, classes)
+    add_methods(ast, methods)
+
+    for klass in ast.classes:
+        klass.create_pointer_class()
+        klass.create_send_method()
+        # TODO: Pandas can't have tags and description because they break the dict
```

Comment:
Q: Why is this?

Reply:
Because it sets attr on them and the keys arent keys for the dataframe or series.